modules/table/src/main/java/org/apache/ignite/internal/table/distributed/StorageUpdateHandler.java

```diff
@@ -150,6 +224,26 @@ public void handleUpdateAll(
         });
     }
 
+    private void executeBatchGc(@Nullable HybridTimestamp newLwm) {
+        if (newLwm != null) {
```

Comment:
How about using the Early Return pattern? (that is, `if (newLwm == null) { return; }` in the very beginning of the method, so the main method logic does not need to be indented)